android/build.gradle

```diff
@@ -1,12 +1,19 @@
+def safeExtGet(prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
 
 buildscript {
```

Comment:
can we move this to if check?

Reply:
Sorry what do you mean?

Reply:
This is the same exact check that's used in **dozens** of projects (I just copy+pasted it) :)

Reply:
I would like to move repositories block inside the if check

Reply:
Ooh, I understand now. Okay sure, let me try.

Reply:
@dulmandakh - I updated it. I agree, it looks a bit cleaner. Thanks!